src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
```

Comment:
```suggestion
export const MoviesList: React.FC<Props> = ({ movies }) => (
```
movies is required referring to your Props type